go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/gnolang/gno
 go 1.20
 
 require (
-	github.com/btcsuite/btcd v0.22.0-beta.0.20220111032746-97732e52810c
-	github.com/btcsuite/btcd/btcutil v1.0.0
+	github.com/btcsuite/btcd/btcutil v1.1.0
 	github.com/btcsuite/btcutil v1.0.2
```

Comment:
can we remove this dependency?

it is only used in one test file (`tm2/pkg/crypto/secp256k1/secp256k1_test.go`), and should hopefully avoid https://github.com/gnolang/gno/issues/1063 from popping up again.

Reply:
removed